dual_net.py

```diff
@@ -210,16 +210,33 @@ def my_res_layer(inputs):
             learning_rate, FLAGS.sgd_momentum).minimize(
                 combined_cost, global_step=global_step)
 
+    policy_target_top_1 = tf.argmax(labels['pi_tensor'], axis=1)
+    policy_output_top_1 = tf.argmax(policy_output, axis=1)
+
+    policy_output_in_top3 = tf.to_float(
+        tf.nn.in_top_k(policy_output, policy_target_top_1, k=3))
+
+    policy_top_1_confidence = tf.reduce_max(policy_output, axis=1)
+    policy_target_top_1_confidence = tf.boolean_mask(
```

Comment:
tf.reduce_max(policy_output, axis=1)

Reply:
discussed offline.

policy_top_1 is how much weight is going to the top policy output (human interpretations is: how confident is policy) policy_target_top_1 measures how much weight is going to the top TARGET policy output (i.e. human interpretations is: how much did we look at the move that was ultimately chosen)